README.md

```diff
@@ -47,6 +47,16 @@ $ cargo build --release
 
 This will create the `lucky_commit` binary (`lucky_commit.exe` on Windows) in the `target/release` directory. You can move this to wherever you want, or set up an alias for it.
 
+#### Distro packages
+
+##### Arch Linux
+
+`lucky-commit` can be installed from the [community repository](https://archlinux.org/packages/community/x86_64/lucky-commit/) using [pacman](https://wiki.archlinux.org/title/Pacman):
```

Comment:
Ideally, I'd like to keep the "Installation" section as close as possible to the "Troubleshooting linker errors" section visually, so that if someone runs the install command and gets an error, they are more likely to see the fallback instructions rather than giving up/assuming things are broken.

To accommodate this, I would prefer if this section was below the "Troubleshooting linker errors" section, or perhaps if it was a bullet point in that section.

Reply:
I pushed 00000000678da6b617932fe9b17b0ce57e397967. Does it look good now?